packages/components/Spinner/src/Spinner.tsx

```diff
@@ -0,0 +1,72 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { NativeModules, ViewProps } from 'react-native';
+import * as React from 'react';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const spinnerName = 'MSFSpinnerView';
+
+const NativeSpinnerView = ensureNativeComponent(spinnerName);
+
+export type Size = 'xSmall' | 'small' | 'medium' | 'large' | 'xLarge';
```

Comment:
It feels like we want to standardize size enums. PersonaCoin has size8, size24, etc. I'm not sure if that or this is the right schema but it's worth starting a discussion in the teams channel about how to make all of these consistent. It seems like the icon component should have the same sizing logic as well.

Reply:
we've purposely don't use any numbers in the enum because designers seems to change the ramp size frequently enough that it doesn't withstand very long. similar to how apple UX enum also don't have numbers because it could change os to os.

Reply:
A size enum sounds great! I don't know if it should be the same for the icon component, but that sounds reasonable.

Fwiw, FluentUI Apple, FluentUI Android, and FluentUI Web  use the same sizes (except apple has the extra size xLarge), so I prefer the strings. Optionally a nice additional API would be if we could specify the size to be any number in points.

https://developer.microsoft.com/en-us/fluentui#/controls/ios/spinner
https://developer.microsoft.com/en-us/fluentui#/controls/android/spinner
https://developer.microsoft.com/en-us/fluentui#/controls/web/spinner

Reply:
I'm following this up inside our Teams Channel (and also the Fluent Community Team). I'll keep this change, and filed #475 to follow up